README.md

```diff
@@ -479,70 +479,44 @@ PRs adding or improving logging facilities are very welcome.
 
 ### Developer Tools
 
-You can get a debugging environment similar to the Chrome developer
-tools running with a FastBoot app, although it's not (yet) as easy as
-in the browser.
+Thanks to recent improvements in NodeJS it is now possible to get a
+debugging environment that you can connect to with Chrome DevTools (version 55+).
+You can find more information on the new debugging method on Node's
+[official documentation](https://nodejs.org/en/docs/inspector/) but here
+is a quick-start guide:
 
-First, install the Node Inspector:
 
-```sh
-npm install node-inspector -g
-```
-
-Make sure you install a recent release; in our experience, older
-versions will segfault when used in conjunction with Contextify, which
-FastBoot uses for sandboxing.
-
-Next, start the inspector server. We found the experience too slow to be
-usable until we discovered the `--no-preload` flag, which waits to
-fetch the source code for a given file until it's actually needed.
-
-```sh
-node-inspector --no-preload
-```
-
-Once the debug server is running, you'll want to start up the FastBoot
-server with Node in debug mode. One thing about debug mode: it makes
-everything much slower.
+First let's start up the FastBoot server with Node in debug mode. One thing
+about debug mode: it makes everything much slower.
 
 ```sh
-ember build && node --debug-brk ./node_modules/.bin/ember serve
+ember build && node --debug-brk --inspect ./node_modules/.bin/ember serve
```

Comment:
I'm not sure exactly why we would want to `ember build &&` here. Lets remove that for now (we can bring it back if someone remembers why it was there and its still important)...